app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+db = SQLAlchemy(model_class=Base)
+
+# create the app
+app = Flask(__name__)
+
+# configure the SQLite database, relative to the app instance folder
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///animeDB"
+
+# initialize the app with the extension
+db.init_app(app)
+
+with app.app_context():
+    db.Model.metadata.reflect(db.engine)
+
+
+class Anime(db.Model):
```

Comment:
Can you put the model definition in a specific file called models?

Reply:
models.py already created in previous PR for creating the DB, it includes Anime model definition.

The class defined at line 25 includes a reflection of a table in the DB.

If it is still better to move this class definition to another file, please suggest a name

Reply:
> models.py already created in previous PR for creating the DB, it includes Anime model definition.

I cannot see the models.py file so wonder if maybe it has not been pushed to the remote.
Here, that's a simple project so in theory putting the model part in the models.py is optional but we want to create a good structure to show your software engineer quality about structuring a project. So let's create a file called models.py to put the model part into it.